src/lib/users/dialogs/inst_user_first.rb

```diff
@@ -0,0 +1,661 @@
+        "<p>\nEnter the <b>User's Full Name</b>, <b>Username</b>, and <b>Password</b> to\n" +
+        "assign to this user account.\n</p>\n") +
+      _(
+        "<p>\nWhen entering a password, distinguish between uppercase and\n" +
+        "lowercase. Passwords should not contain any accented characters or umlauts.\n</p>\n"
+      )
+
+      help += (
+        # TRANSLATORS: %{min} and %{max} will be replaced by numbers
+        _(
+          "<p>\nThe password length should be between %{min}\n and %{max} characters.\n</p>\n"
+        ) %
+        {
+          min: UsersSimple.GetMinPasswordLength("local"),
+          max: UsersSimple.GetMaxPasswordLength("local")
+        }
+      ) + UsersSimple.ValidPasswordHelptext
+
+      help += ::Users::CAPasswordValidator.new.help_text
+
+      help += _(
+        "<p>\nTo ensure that the password was entered correctly,\n" \
+        "repeat it exactly in a second field. Do not forget your password.\n" \
+        "</p>\n"
+      ) +
+      _(
+        "<p>\nFor the <b>Username</b> use only letters (no accented characters), digits, and <tt>._-</tt>.\n" \
+        "Do not use uppercase letters in this entry unless you know what you are doing.\n" \
+        "Usernames have stricter restrictions than passwords. You can redefine the\n" \
+        "restrictions in the /etc/login.defs file. Read its man page for information.\n" \
+        "</p>\n"
+      ) +
+      _(
+        "<p>Check <b>Use this password for system administrator</b> if the " \
+        "same password as entered for the first user should be used for root.</p>"
+      ) +
+      _(
+        "<p>\nThe username and password created here are needed to log in " \
+        "and work with your Linux system. With <b>Automatic Login</b> enabled, " \
+        "the login procedure is skipped. This user is logged in automatically.</p>\n"
+      ) +
+      _(
+        "<p>\nHave mail for root forwarded to this user by checking <b>Receive System Mail</b>.</p>\n"
+      )
+
+      if import_available?
+        help += "<p>\n<b>" + _("Import User Data from a Previous Installation") + "</b>\n</p>\n"
+        help += _(
+          "<p>\nA previous Linux installation with local users has been detected.\n" \
+          "The information there can be used to create users in the system being installed.\n" \
+          "Use the <b>Choose Users</b> button to select some users. Their basic information will\n" \
+          "be imported.\n</p>\n"
+        )
+      end
+
+      help += "<p>\n<b>" + _("Skip User Creation") + "</b>\n</p>\n"
+      help += _(
+        "<p>\nSometimes root is the only needed local user, like in network environments\n" \
+        "with an authentication server. Select this option to proceed without creating\n" \
+        "a local user.\n</p>\n"
+      )
+
+      help
+    end
+
+  private
+
+    # Imports all the used YaST modules
+    #
+    # Importing them before class initialization has been observed to
+    # be problematic
```

Comment:
Quite random exceptions raised in `import`. Usually (but not always) while trying to import `Report`.